### PR TITLE
vendor: update ct to 0.8.0

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: platform/conf/conf.go [hunk 1]
```diff
@@ -232,7 +232,7 @@ func (u *UserData) Render(ctPlatform string) (*Conf, error) {
 			plog.Warningf("rendering Container Linux config: %s", report)
 		}
 
-		c.ignitionV21 = &ignc
```
Comment (ajeddeloh) [inline platform/conf/conf.go]: I'd break this into a seperate commit, otherwise LGTM

Comment (bgilbert) [reply platform/conf/conf.go]: Yeah, that'd be my preference, but I didn't want to break the build for one commit.

Comment (ajeddeloh) [reply platform/conf/conf.go]: At least call it out in the commit message?

Comment (bgilbert) [reply platform/conf/conf.go]: Done.

**Pull request merged**